system_files/desktop/shared/usr/lib/systemd/system/ryzenadj_coper.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=Per-Core CPU Undervolt for Steam Deck
+
+[Service]
+Type=oneshot
+EnvironmentFile=-/etc/default/%p
+ExecStartPre=/usr/bin/sleep 10
```

Use `system_files/desktop/shared/usr/libexec/hwsupport/valve-hardware` to check the device is a steam-deck
```suggestion
ExecCondition=/usr/libexec/hwsupport/valve-hardware
```
```suggestion